app/views/devise/registrations/new_with_student.html.erb

```diff
@@ -27,133 +27,135 @@
+                  <%= f.label :Desc_estatus_solicitud %>
+                  <%= f.date_field :desc_request_status, :value => Time.now.strftime("%Y-%m-%d")  %>
                 </div>
-                <div class="col s2 input-field inline">
-                  <%= c.label :Celular %>
-                  <%= c.telephone_field :cellphone_number %>
+                  <div class="col s2 input-field inline">
+                    <%= c.label :Teléfono %>
+                    <%= c.telephone_field :phone_number %>
+                  </div>
+                  <div class="col s2 input-field inline">
+                    <%= c.label :Celular %>
+                    <%= c.telephone_field :cellphone_number %>
+                  </div>
+              </div>
+              <h5>
+                Domicilio:
+              </h5>
+              <div class="row">
+                <div class="col s3 input-field inline">
+                  <%= c.label :Calle_de_Domicilio,:style=>'color:red;' %>
+                  <%= c.text_field :street_address, :required => true  %>
                 </div>
-            </div>
+                <div class="col s3 input-field inline">
+                  <%= c.label :Número_exterior%>
+                  <%= c.number_field :street_number_address_ext %>
+                </div>
+                <div class="col s3 input-field inline">
+                  <%= c.label :Número_interior %>
+                  <%= c.number_field :street_number_address_int %>
+                </div>
+                <div class="col s3 input-field inline">
+                  <%= c.label :Colonia %>
+                  <%= c.text_field :colony_address %>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col s4 input-field inline">
+                  <%= c.label :Ciudad,:style=>'color:red;'%>
+                  <%= c.text_field :city_address, :required => true  %>
+                </div>
+                <div class="col s4 input-field inline">
+                  <%= c.label :Municipio %>
+                  <%= c.text_field :municipiality_address %>
+                </div>
+                <div class="col s4 input-field inline">
+                  <%= c.label :Estado %>
+                  <%= c.text_field :state_address %>
+                </div>
+              </div>
+            <% end %>
             <h5>
-              Domicilio:
+              Información de Estudios:
             </h5>
+            <%= s.fields_for :scholarship_attributes do |sch| %>
             <div class="row">
               <div class="col s3 input-field inline">
-                <%= c.label :Calle_de_Domicilio,:style=>'color:red;' %>
-                <%= c.text_field :street_address, :required => true  %>
+                <%= select_tag 'scholarship_oportunity_id', options_for_select(ScholarshipOportunity.all.collect {|p| [ p.name, p.id ] }.insert(0, "Convocatoria")),:style=>'color:red;', :required => true ,class: "browser-default"%>
```

Comment:
Does this work? Would it allow the user to select Convocatoria as an option?

Reply:
Yes (that was already there, I just fixed the formatting).